### DISCUSSION TITLE
Releases Not Published to NPM?

### DISCUSSION THREAD

**Discussion by brianlenz** [dooboolab/react-native-audio-recorder-player#618] [General]
Hello! First and foremost, thanks for the awesome library 💪 I happened to notice that the last two releases, [3.6.9](https://github.com/hyochan/react-native-audio-recorder-player/releases/tag/3.6.9) and [3.6.10](https://github.com/hyochan/react-native-audio-recorder-player/releases/tag/3.6.10), haven't been published to NPM. Is this expected?

**Comment by hyochan** [comment]
Thanks for reminder! I am not sure why this is keep failing https://github.com/hyochan/react-native-audio-recorder-player/actions/runs/9590556784.

I just published package this manually for this time!

**Comment by brianlenz** [comment]
@hyochan I was just about to come suggest updating the `setup-node` version, but I see you already did in 83dc82896178cdfaaadda77802bc49f54ae53e68 🚀  I'm guessing that should do it! 💪 

Thanks again!
